teuthology/orchestra/run.py

```diff
@@ -91,7 +91,7 @@ def execute(self):
         if self.label:
             prefix = "Running ({label}):".format(label=self.label)
         log.getChild(self.hostname).info(u"{prefix} {cmd!r}".format(
-            cmd=self.command, prefix=prefix))
+            cmd=self.command.strip().replace('\n', ' '), prefix=prefix))
```

Comment:
I thinks replacing the newlines has two disadvantages, this breaks bash syntax, it is not that readable as multiline.
So I vote for logging multiline commands as multiline logs commands.
From the other side, it is not necessary to prefix each line with 'Running (label) prefix' and it should be only logged once before the script.

Reply:
@kshtsk agreed. i incorporated proposed change.